projects/kit/components/input-date-range/input-date-range.directive.ts

```diff
@@ -18,5 +18,6 @@ export class TuiInputDateRangeDirective extends TuiAbstractTextfieldHost<TuiInpu
 
     process(input: HTMLInputElement): void {
         input.inputMode = `numeric`;
+        input.placeholder = this.host.computedExampleText;
```

Comment:
I'll state this once but it might be relevant elsewhere:
There are two cases where we set native attributes through those directives
1. Initial value, that people might want to override 
2. Enforced values, required by this type of control

We gotta think about this in general 🤔